### ISSUE TITLE
No description on how to use the tool

### ISSUE THREAD

**Issue by christian-rli** [rl-institut/WAM_APP_FRED#66]
There should be short descriptions in every tab.

**Comment by jh-RLI** [comment]
@Bachibouzouk in time series tab the description mentions to select a variable. I think its not quit clear which variable we mean, is it?

**Comment by jh-RLI** [comment]
@christian-rli @Bachibouzouk We could implement a dialog window thats pops up when a user first opens the app. There could be a short summary about the app and the three components. This would help the user to start working with the app. 

This can be done with modal jquery ui dialog, I tryed to implement this but for some reason its not working and now i have no time left. 

This would be a demo of what i mean:
Save the following to a file named like dialog.html
```
<!DOCTYPE html>
<html>
  <head><!--Meta Tags, Titel, etc. -->
    <title>jQuery UI: Widget (Dialog)</title>
    <meta charset="utf-8">
	<link rel="stylesheet" href="http://code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css">
    <script src="http://code.jquery.com/jquery-1.9.1.js"></script>
    <script src="http://code.jquery.com/ui/1.10.3/jquery-ui.js"></script>
	<!--Eigene Skripte bzw. von jQuery oder jQuery UI abhängige Skripte-->
    <script>
	  $(function() {
		$( "#dialog-message" ).dialog({
		  modal: true,
		  buttons: {
			Ok: function() {
			  $( this ).dialog( "close" );
			}
		  }
		});
	  });
    </script>
  </head>
  <body>
	<h1>UI-Möglichkeiten: Widgets</h1>
	<h2>dialog() - Message</h2>
	<p>Hier ist ein Absatz</p>
	<div id="dialog-message" title="Download fertig!">
	  <p>
		<span class="ui-icon ui-icon-circle-check" style="float:left; margin:0 7px 50px 0;"></span>
		Die Datei wurde <b>vollständig</b> heruntergeladen.
	  </p>
	</div>
	<p>Hier ist noch ein Absatz!</p>
  </body>
</html>
```

**Comment by nesnoj** [comment]
You may want to use WAM's [InfoButton](https://github.com/rl-institut/WAM/blob/ba2d4b9d93a4fcd4b4e203a5d9b1527059ab8bf5/utils/widgets.py#L36) widget which reads a markdown file (nice & easy content production), converts to HTML and displays the content in a reveal window. Window can also be triggered via JS.

Have a look into StEmp ABW where I use it as welcome window shown on startup and can be re-opened using a button on the nav bar:
- [md file](https://github.com/rl-institut/WAM_APP_stemp_abw/blob/8872cc34d3066473653a73f5ed6938ee045d56eb/config/reveals/welcome.md)
- [init on startup](https://github.com/rl-institut/WAM_APP_stemp_abw/blob/8872cc34d3066473653a73f5ed6938ee045d56eb/config/prepare_texts.py#L10)
- [template context](https://github.com/rl-institut/WAM_APP_stemp_abw/blob/8872cc34d3066473653a73f5ed6938ee045d56eb/views/__init__.py#L92)
- [in nav bar](https://github.com/rl-institut/WAM_APP_stemp_abw/blob/8872cc34d3066473653a73f5ed6938ee045d56eb/templates/stemp_abw/navigation_base.html#L16)

**Comment by christian-rli** [comment]
Thanks for the hint @nesnoj ! Going to add this in my next long bus ride :)

**Comment by christian-rli** [comment]
Thank you @Bachibouzouk for taking this over. I now have a prepared welcome text that I can adapt to any markup language once a welcome window is implemented.
[welcome.txt](https://github.com/rl-institut/WAM_APP_FRED/files/3445694/welcome.txt)

**Comment by christian-rli** [comment]
... implementing the button was a bit more fiddly than expected. I created a new welcome tab which has the same functionality without adding another navigation element, albeit being less fancy.

**Comment by christian-rli** [comment]
addressed in #125
